fnv_test.go

```diff
@@ -1,8 +1,9 @@
 package bigcache
 
 import (
-	"hash/fnv"
 	"testing"
+
+	"github.com/pierrec/xxHash/xxHash64"
```

Comment:
But in code there is another package 🤔 
https://github.com/allegro/bigcache/pull/136/files#diff-05f222304907b8cb5a43cc60c71cdf9dR4